client/signup/config/step-components.js

```diff
@@ -61,7 +61,7 @@ const stepNameToModuleName = {
 export async function getStepComponent( stepName ) {
 	const moduleName = stepNameToModuleName[ stepName ];
 	const module = await import(
-		/* webpackChunkName: "async-load-signup-steps-[request]", webpackInclude: /signup\/steps\/[a-z-]+\/index.jsx$/ */ `signup/steps/${ moduleName }`
+		/* webpackChunkName: "async-load-signup-steps-[request]", webpackInclude: /signup\/steps\/[a-z-]+\/index.(jsx|tsx)$/ */ `signup/steps/${ moduleName }`
```

Comment:
Was this change required? Doesn't look like any `tsx` was added here.

Reply:
Good spotting @sirreal . I did introduce a new [TS component](https://github.com/Automattic/wp-calypso/pull/34768/commits/d1ebdbc74bb9b36a4e3ac81115ce3d51cb416df8#diff-04ea2f1da94759c5b4fdd338edf0e57f), but ditched it due to reasons. Nothing to do with TS itself.